[Sat Jan 11 23:00:24 UTC 2025][addStream@logging.cc:107                                     | ---] Successfully added log stream. Stream type: JSON FOG stream
[init@components_list_impl.h:126                              | ---] Initializing component: Environment
[init@components_list_impl.h:126                              | ---] Initializing component: DebugIS
[2025-01-11T23:00:24.340761: : init@components_list_impl.h:126                            | ---] Initializing component: Version
[2025-01-11T23:00:24.342153: : init@components_list_impl.h:126                            | ---] Initializing component: Buffer
[2025-01-11T23:00:24.342919: : init@components_list_impl.h:126                            | ---] Initializing component: ShellCmd
[2025-01-11T23:00:24.344294: : init@components_list_impl.h:126                            | ---] Initializing component: GenericMetric
[2025-01-11T23:00:24.345000: : init@components_list_impl.h:126                            | ---] Initializing component: Messaging
[2025-01-11T23:00:24.358809: : init@components_list_impl.h:126                            | ---] Initializing component: MainloopComponent
[2025-01-11T23:00:24.359664: : init@components_list_impl.h:126                            | ---] Initializing component: ConfigComponent
[2025-01-11T23:00:24.360272: : init@components_list_impl.h:126                            | ---] Initializing component: InstanceAwareness
[2025-01-11T23:00:24.360663: : init@components_list_impl.h:126                            | ---] Initializing component: IntelligenceComponentV2
[2025-01-11T23:00:24.386486: : init@components_list_impl.h:126                            | ---] Initializing component: AgentDetails
[2025-01-11T23:00:24.440422: : getMachineTypeFromDmiTable@agent_details.cc:354            | ###] Error. Could not decode the DMI table. Table value is empty
[2025-01-11T23:00:24.456034: : registerMachineType@agent_details.cc:376                   | ###] Error. Machine type is unrecognized
[2025-01-11T23:00:24.458447: : registerMachineType@agent_details.cc:381                   | ---] Setting machine type 3
[2025-01-11T23:00:24.471062: : init@components_list_impl.h:126                            | ---] Initializing component: LoggingComp
[2025-01-11T23:00:24.473731: : init@components_list_impl.h:126                            | ---] Initializing component: TimeProxyComponent
[2025-01-11T23:00:24.475571: : init@components_list_impl.h:126                            | ---] Initializing component: SignalHandler
[2025-01-11T23:00:24.476415: : init@components_list_impl.h:126                            | ---] Initializing component: RestServer
[2025-01-11T23:00:24.477371: : init@components_list_impl.h:126                            | ---] Initializing component: Encryptor
[2025-01-11T23:00:24.480403: : init@components_list_impl.h:126                            | ---] Initializing component: SocketIS
[2025-01-11T23:00:24.484301: : init@components_list_impl.h:126                            | ---] Initializing component: CPUCalculator
[2025-01-11T23:00:24.489349: : init@components_list_impl.h:126                            | ---] Initializing component: CPUManager
[2025-01-11T23:00:24.495163: : init@components_list_impl.h:126                            | ---] Initializing component: MemoryCalculator
[2025-01-11T23:00:24.506400: : init@components_list_impl.h:126                            | ---] Initializing component: TenantManager
[2025-01-11T23:00:24.508222: : init@components_list_impl.h:126                            | ---] Initializing component: GenericRulebase
[2025-01-11T23:00:24.509011: : init@components_list_impl.h:126                            | ---] Initializing component: Table
[2025-01-11T23:00:24.511258: : init@components_list_impl.h:126                            | ---] Initializing component: NginxAttachment
[2025-01-11T23:00:24.513677: : init@nginx_attachment.cc:263                               | ---] Successfully initialized NGINX Attachment
[2025-01-11T23:00:24.515946: : init@components_list_impl.h:126                            | ---] Initializing component: GradualDeployment
[2025-01-11T23:00:24.524579: : init@components_list_impl.h:126                            | ---] Initializing component: HttpManager
[2025-01-11T23:00:24.529471: : init@components_list_impl.h:126                            | ---] Initializing component: Layer-7 Access Control
[2025-01-11T23:00:24.532274: : init@components_list_impl.h:126                            | ---] Initializing component: RateLimit
[2025-01-11T23:00:24.536997: : init@components_list_impl.h:126                            | ---] Initializing component: WaapComponent
[2025-01-11T23:00:26.057354: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ScannerDetector'
[2025-01-11T23:00:26.924364: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-11T23:00:26.926120: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-11T23:00:26.928225: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-11T23:00:26.929921: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-11T23:00:26.967912: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-ab.js, static resource's path: /etc/cp/conf/waap/cp-ab.js
[2025-01-11T23:00:26.975129: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-csrf.js, static resource's path: /etc/cp/conf/waap/cp-csrf.js
[2025-01-11T23:00:26.976081: : init@components_list_impl.h:126                            | ---] Initializing component: IPSComp
[2025-01-11T23:00:26.977158: : init@components_list_impl.h:126                            | ---] Initializing component: KeywordComp
[2025-01-11T23:00:26.978174: : init@components_list_impl.h:126                            | ---] Initializing component: GeoLocation
[2025-01-11T23:00:26.980238: : initGeoDbObj@geo_location.cc:109                           | ###] maxMindDB error: Error opening the specified MaxMind DB file
[2025-01-11T23:00:26.981199: : init@components_list_impl.h:126                            | ---] Initializing component: HttpGeoFilter
[2025-01-11T23:00:26.981977: : run@mainloop.cc:236                                        | ---] Starting the Mainloop
[2025-01-11T23:00:26.982683: : <14> operator()@rest_server.cc:210                         | ---] REST server started: 12003. Accepting: loopback connections
[2025-01-11T23:00:27.010334: : <5> reportStartupEvent@mainloop.cc:224                     | ---] Startup report was successfully sent to fog
[2025-01-11T23:00:27.011258: : <10> closeLogFile@file_stream.cc:119                       | ###] Failed in closing log file. File path: 5
[2025-01-11T23:00:29.177852: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-11T23:00:40.693230: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-11T23:01:05.246667: : <37> load@triggers_config.cc:205                           | ###] Failed to parse the extendLoggingMinSeverityfield: 'high'
[2025-01-11T23:01:05.248873: : <37> addStream@logging.cc:107                              | ---] Successfully added log stream. Stream type: JSON File stream
[2025-01-11T23:01:05.261776: : <37> loadUserLimitsPolicy@WaapConfigBase.cc:282            | ---] [USER LIMITS] policy loaded:
[Policy] urlMaxSize: 32768  httpHeaderMaxSize: 102400  httpBodyMaxSize: 1024000000  maxObjectDepth: 40  httpIllegalMethodsAllowed: false
[2025-01-11T23:01:05.275327: : <37> loadUserLimitsPolicy@WaapConfigBase.cc:282            | ---] [USER LIMITS] policy loaded:
[Policy] urlMaxSize: 32768  httpHeaderMaxSize: 102400  httpBodyMaxSize: 1024000000  maxObjectDepth: 40  httpIllegalMethodsAllowed: false
[2025-01-11T23:01:05.294788: : <37> closeLogFile@file_stream.cc:119                       | ###] Failed in closing log file. File path: /var/log/nano_agent/cp-nano-http-transaction-handler.log5
[2025-01-11T23:01:05.298248: : <37> addStream@logging.cc:107                              | ---] Successfully added log stream. Stream type: JSON FOG stream
[2025-01-11T23:01:05.303437: : <37> initGeoDbObj@geo_location.cc:109                      | ###] maxMindDB error: Error opening the specified MaxMind DB file
[2025-01-11T23:02:27.882607: : <25> registerAttachmentProcess@nginx_attachment.cc:540     | ---] Successfully registered attachment
[2025-01-11T23:02:27.892805: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:27.922751: : <39> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ScannerDetector'
[2025-01-11T23:02:27.924846: : <39> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-11T23:02:27.926746: : <39> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-11T23:02:27.928834: : <39> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-11T23:02:27.932325: : <39> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-11T23:02:28.004752: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:28.006646: : <39> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-11T23:02:28.017752: : <41> getSharedStorageHost@TuningDecision.cc:184            | ---] shared storage host is set to appsec-shared-storage
[2025-01-11T23:02:28.030686: : <41> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-11T23:02:28.043787: : <41> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-11T23:02:28.044666: : <41> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-11T23:02:28.045629: : <41> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-11T23:02:28.046501: : <41> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-11T23:02:28.047194: : <41> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-11T23:02:28.074227: : <50> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-11T23:02:28.086319: : <50> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-11T23:02:28.095493: : <50> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-11T23:02:28.096398: : <50> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-11T23:02:28.103391: : <50> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-11T23:02:28.104113: : <50> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-11T23:02:28.105673: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -1, error response: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-11T23:02:28.109904: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.112237: : <52> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.113761: : <53> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.114603: : <54> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.116005: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:28.134798: : <43> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.135933: : <43> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-11T23:02:28.136911: : <43> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.137997: : <43> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.139053: : <43> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-11T23:02:28.140079: : <43> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-11T23:02:28.148945: : <45> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.152378: : <45> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-11T23:02:28.173397: : <45> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.196011: : <45> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.203269: : <45> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-11T23:02:28.223417: : <47> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.239215: : <47> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-11T23:02:28.257436: : <47> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.288383: : <47> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.297743: : <47> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-11T23:02:28.299685: : <47> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-11T23:02:28.309126: : <49> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.313639: : <49> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-11T23:02:28.315873: : <49> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.322902: : <49> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-11T23:02:28.325891: : <49> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-11T23:02:28.546778: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:28.565980: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:29.013296: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-11T23:02:29.184445: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-11T23:02:33.391858: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:34.305592: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-11T23:02:36.508566: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.522021: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.657673: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.682964: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.701348: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.843726: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.862813: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:36.916510: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:37.190213: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:37.373606: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:37.442915: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:37.463678: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:38.264621: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:38.572398: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:41.542312: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:48.816866: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:48.872634: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:48.902222: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.057463: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.414423: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.766499: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.807468: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.910874: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:49.933314: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:50.043710: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:50.066071: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:50.592285: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[2025-01-11T23:02:50.858299: : <39> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.22.0.1, Error: Maxmind db is uninitialized
[Sun Jan 12 09:57:22 UTC 2025][addStream@logging.cc:107                                     | ---] Successfully added log stream. Stream type: JSON FOG stream
[init@components_list_impl.h:126                              | ---] Initializing component: Environment
[init@components_list_impl.h:126                              | ---] Initializing component: DebugIS
[2025-01-12T09:57:23.093806: : init@components_list_impl.h:126                            | ---] Initializing component: Version
[2025-01-12T09:57:23.095353: : init@components_list_impl.h:126                            | ---] Initializing component: Buffer
[2025-01-12T09:57:23.096419: : init@components_list_impl.h:126                            | ---] Initializing component: ShellCmd
[2025-01-12T09:57:23.097237: : init@components_list_impl.h:126                            | ---] Initializing component: GenericMetric
[2025-01-12T09:57:23.102675: : init@components_list_impl.h:126                            | ---] Initializing component: Messaging
[2025-01-12T09:57:23.112130: : init@components_list_impl.h:126                            | ---] Initializing component: MainloopComponent
[2025-01-12T09:57:23.114660: : init@components_list_impl.h:126                            | ---] Initializing component: ConfigComponent
[2025-01-12T09:57:23.117152: : init@components_list_impl.h:126                            | ---] Initializing component: InstanceAwareness
[2025-01-12T09:57:23.118772: : init@components_list_impl.h:126                            | ---] Initializing component: IntelligenceComponentV2
[2025-01-12T09:57:23.119558: : init@components_list_impl.h:126                            | ---] Initializing component: AgentDetails
[2025-01-12T09:57:23.142431: : getMachineTypeFromDmiTable@agent_details.cc:354            | ###] Error. Could not decode the DMI table. Table value is empty
[2025-01-12T09:57:23.145392: : registerMachineType@agent_details.cc:376                   | ###] Error. Machine type is unrecognized
[2025-01-12T09:57:23.155844: : registerMachineType@agent_details.cc:381                   | ---] Setting machine type 3
[2025-01-12T09:57:23.171000: : init@components_list_impl.h:126                            | ---] Initializing component: LoggingComp
[2025-01-12T09:57:23.173803: : init@components_list_impl.h:126                            | ---] Initializing component: TimeProxyComponent
[2025-01-12T09:57:23.177745: : init@components_list_impl.h:126                            | ---] Initializing component: SignalHandler
[2025-01-12T09:57:23.182313: : init@components_list_impl.h:126                            | ---] Initializing component: RestServer
[2025-01-12T09:57:23.183313: : init@components_list_impl.h:126                            | ---] Initializing component: Encryptor
[2025-01-12T09:57:23.185995: : init@components_list_impl.h:126                            | ---] Initializing component: SocketIS
[2025-01-12T09:57:23.189129: : init@components_list_impl.h:126                            | ---] Initializing component: CPUCalculator
[2025-01-12T09:57:23.191366: : init@components_list_impl.h:126                            | ---] Initializing component: CPUManager
[2025-01-12T09:57:23.201826: : init@components_list_impl.h:126                            | ---] Initializing component: MemoryCalculator
[2025-01-12T09:57:23.204029: : init@components_list_impl.h:126                            | ---] Initializing component: TenantManager
[2025-01-12T09:57:23.205929: : init@components_list_impl.h:126                            | ---] Initializing component: GenericRulebase
[2025-01-12T09:57:23.214154: : init@components_list_impl.h:126                            | ---] Initializing component: Table
[2025-01-12T09:57:23.217649: : init@components_list_impl.h:126                            | ---] Initializing component: NginxAttachment
[2025-01-12T09:57:23.222366: : init@nginx_attachment.cc:263                               | ---] Successfully initialized NGINX Attachment
[2025-01-12T09:57:23.224121: : init@components_list_impl.h:126                            | ---] Initializing component: GradualDeployment
[2025-01-12T09:57:23.225352: : init@components_list_impl.h:126                            | ---] Initializing component: HttpManager
[2025-01-12T09:57:23.226540: : init@components_list_impl.h:126                            | ---] Initializing component: Layer-7 Access Control
[2025-01-12T09:57:23.239575: : init@components_list_impl.h:126                            | ---] Initializing component: RateLimit
[2025-01-12T09:57:23.240513: : init@components_list_impl.h:126                            | ---] Initializing component: WaapComponent
[2025-01-12T09:57:24.841583: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ScannerDetector'
[2025-01-12T09:57:24.844498: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-12T09:57:24.846473: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T09:57:24.848919: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-12T09:57:24.851185: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T09:57:24.887651: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-ab.js, static resource's path: /etc/cp/conf/waap/cp-ab.js
[2025-01-12T09:57:24.889654: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-csrf.js, static resource's path: /etc/cp/conf/waap/cp-csrf.js
[2025-01-12T09:57:24.890264: : init@components_list_impl.h:126                            | ---] Initializing component: IPSComp
[2025-01-12T09:57:24.906821: : init@components_list_impl.h:126                            | ---] Initializing component: KeywordComp
[2025-01-12T09:57:24.908765: : init@components_list_impl.h:126                            | ---] Initializing component: GeoLocation
[2025-01-12T09:57:24.910882: : initGeoDbObj@geo_location.cc:109                           | ###] maxMindDB error: Error opening the specified MaxMind DB file
[2025-01-12T09:57:24.911946: : init@components_list_impl.h:126                            | ---] Initializing component: HttpGeoFilter
[2025-01-12T09:57:24.912813: : run@mainloop.cc:236                                        | ---] Starting the Mainloop
[2025-01-12T09:57:24.914620: : <14> operator()@rest_server.cc:210                         | ---] REST server started: 12003. Accepting: loopback connections
[2025-01-12T09:57:24.942547: : <5> reportStartupEvent@mainloop.cc:224                     | ---] Startup report was successfully sent to fog
[2025-01-12T09:57:24.943765: : <10> closeLogFile@file_stream.cc:119                       | ###] Failed in closing log file. File path: 5
[2025-01-12T09:57:27.096896: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T09:57:28.548133: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T09:59:59.903130: : <29> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='ScannerDetector' last modified state: 
[2025-01-12T09:59:59.953615: : <30> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='KeywordIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T10:00:00.009449: : <30> operator()@Serializator.cc:570                        | !!!] syncWorker execution time (owner='KeywordIndicatorFilter/ConfidenceCalculator', assetId='') is 0 seconds, too long to cause negative remainingTime. Waiting 0 seconds...
[2025-01-12T10:00:00.011259: : <31> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='TrustedSourcesConfidenceCalculator' last modified state: 
[2025-01-12T10:00:00.028624: : <32> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='TypeIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T10:00:00.039484: : <33> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='TrustedSourcesConfidenceCalculator' last modified state: 
[2025-01-12T10:00:00.067294: : <29> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='ScannerDetector' last modified state: 
[2025-01-12T10:00:00.074420: : <30> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='KeywordIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T10:00:25.270932: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:07:26.010259: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:07:26.185877: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:08:11.346463: : <25> registerAttachmentProcess@nginx_attachment.cc:540     | ---] Successfully registered attachment
[2025-01-12T10:08:11.348773: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:08:11.359499: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ScannerDetector'
[2025-01-12T10:08:11.362061: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-12T10:08:11.364551: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T10:08:11.366629: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-12T10:08:11.369337: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T10:08:11.490001: : <36> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-12T10:08:11.491634: : <38> getSharedStorageHost@TuningDecision.cc:184            | ---] shared storage host is set to appsec-shared-storage
[2025-01-12T10:08:11.501880: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:08:11.502615: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:08:11.515473: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:08:11.516187: : <38> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:08:11.516808: : <38> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:08:11.517605: : <38> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:08:11.518362: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.518885: : <40> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T10:08:11.519496: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.520341: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.521456: : <40> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T10:08:11.523305: : <40> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-12T10:08:11.524274: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.524973: : <42> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T10:08:11.525642: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.526395: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.527838: : <42> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T10:08:11.528563: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.529303: : <44> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T10:08:11.529986: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.530627: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.531623: : <44> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T10:08:11.532452: : <44> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-12T10:08:11.533212: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.533766: : <46> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T10:08:11.535425: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.536368: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.537342: : <46> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T10:08:11.547787: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:08:11.548743: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:08:11.549618: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:08:11.550400: : <47> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:08:11.551014: : <47> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:08:11.551671: : <47> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:08:11.553496: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -1, error response: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:08:11.555996: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.557293: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.558184: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.559348: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:08:11.577510: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:08:12.932563: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:08:25.065263: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:08:25.232689: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:09:45.715264: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:47.994536: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:48.566670: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:48.934038: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:49.875862: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:50.112690: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:50.374299: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:50.631850: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:50.805040: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:51.552782: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:52.211652: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:53.068034: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:09:53.319757: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:51.362532: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:51.676166: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:52.090284: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:52.808650: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:53.151938: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:10:53.415865: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:13:36.174340: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.835879: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 11
[2025-01-12T10:14:54.841417: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 30
[2025-01-12T10:14:54.845400: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 48
[2025-01-12T10:14:54.845911: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 49
[2025-01-12T10:14:54.847134: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 50
[2025-01-12T10:14:54.849341: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 51
[2025-01-12T10:14:54.851785: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.867691: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.889082: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.897261: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.923299: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:54.991014: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:55.061042: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:55.129656: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:55.161037: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:55.225584: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.016354: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.171027: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.175472: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.226413: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.282676: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:14:58.307892: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:17.782697: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:18.709925: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:19.185372: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:19.397577: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:20.160960: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:20.200403: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.242707: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.521028: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.526373: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.598961: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.721200: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:21.991591: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:22.098306: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:22.123965: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:22.236193: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:35.718060: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:39.322502: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:39.752443: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:39.788778: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:40.868850: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:41.289552: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:41.408615: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:51.205240: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:51.754641: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:52.425865: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:52.701545: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:15:52.771498: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:47.974524: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:48.312168: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:48.753108: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:53.760964: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:54.289701: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:54.351499: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:56.855291: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:57.127716: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:57.757586: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:59.131541: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:16:59.402585: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:00.015094: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:00.180152: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:02.490587: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:02.677876: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:02.984109: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:03.216669: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:03.445489: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:03.712588: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:04.031903: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:04.410680: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:04.981805: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:17:26.051641: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:17:26.216847: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:18:11.100391: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:18:11.101889: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:18:11.102606: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:18:11.103217: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:18:11.135683: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:18:11.136607: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:18:11.137354: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:18:11.138142: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:18:11.138691: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:18:11.139701: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:18:11.140516: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:18:11.141400: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:18:11.142340: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:19:59.505541: : <43> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='TypeIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T10:19:59.506785: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:19:59.507220: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T10:20:00.749991: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:01.314759: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:01.433448: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:04.496439: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:20:04.497339: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T10:20:04.632360: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:05.057148: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:05.083528: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:05.822734: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:05.832859: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:06.066843: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:06.485394: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:06.525640: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:09.501503: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:20:09.502629: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T10:20:14.513115: : <43> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/window_20100_9/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T10:20:14.514030: : <43> postData@ConfidenceCalculator.cc:143                  | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/window_20100_9/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T10:20:26.304727: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:20:26.568198: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:20:39.984740: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:40.368644: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:43.461640: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:43.747811: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:43.781472: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:59.638760: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:20:59.892369: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:21:00.159386: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:21:00.352580: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:45.429193: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:45.434350: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:45.747814: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:45.771868: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:45.810793: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:52.563418: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:52.567621: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:53.726584: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:53.742089: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:53.799180: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:54.423172: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:54.439821: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:22:54.568295: : <36> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-12T10:22:54.650600: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:02.996704: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.154743: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.251498: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.332459: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.349548: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.370011: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.416150: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.529043: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.637330: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.694031: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:03.815406: : <36> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-12T10:23:07.673754: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.052192: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.056890: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.061227: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.086877: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.102069: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.135085: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.145515: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.165313: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.192825: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.209033: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.220201: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.234707: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.247679: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.281054: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.347033: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:23:09.368679: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:27:25.325635: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:27:25.629098: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:28:10.681306: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:28:10.682810: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:28:10.683645: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:28:10.684330: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:28:10.719490: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:28:10.720455: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:28:10.721002: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:28:10.721661: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:28:10.722220: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:28:10.723298: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:28:10.724059: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:28:10.724678: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:28:10.725303: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:37:24.497964: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:37:24.696411: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:37:42.706716: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 11
[2025-01-12T10:37:42.707471: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 30
[2025-01-12T10:37:42.708151: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 48
[2025-01-12T10:37:42.708862: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 49
[2025-01-12T10:37:42.709881: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 50
[2025-01-12T10:37:42.710674: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 51
[2025-01-12T10:37:42.712038: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.725322: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.805392: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.825074: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.887945: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.907943: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.973249: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:42.980171: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.043804: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.049826: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.065182: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.083964: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.097177: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.105033: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.119567: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.124990: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.161122: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.211139: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.277608: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.301828: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.326183: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.362324: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.394331: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.443103: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.474322: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.521642: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.543861: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.571052: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.578823: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.625571: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.630007: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.649953: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.656493: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.672275: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.684788: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.706179: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.711141: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.759822: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:43.792306: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.147436: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.444767: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.450140: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.471955: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.483280: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.541164: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.578973: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.634652: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:37:44.657951: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:38:10.258026: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:38:10.259059: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:38:10.259955: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:38:10.260531: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:38:10.292891: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:38:10.294153: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:38:10.294899: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:38:10.295819: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:38:10.296669: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:38:10.297733: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:38:10.298721: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:38:10.299776: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:38:10.300910: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:41:13.718439: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.737395: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.783655: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.833614: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.849609: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.859632: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.877669: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.931019: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.942736: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.960395: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:13.988748: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.015731: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.104844: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.274421: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.288362: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.311636: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.339498: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.347794: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.366912: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.406249: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.439620: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.454150: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.464662: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.476590: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.492205: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.502894: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:14.552462: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:15.238728: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:15.255666: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:41:15.334660: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:47:23.503585: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:47:23.783199: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:48:09.842311: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:48:09.843596: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:48:09.844286: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:48:09.844775: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:48:09.875824: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:48:09.881839: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:48:09.882516: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:48:09.883074: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:48:09.883644: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:48:09.884487: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:48:09.885082: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:48:09.885980: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:48:09.886648: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:53:43.063795: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 11
[2025-01-12T10:53:43.068152: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 30
[2025-01-12T10:53:43.068981: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 48
[2025-01-12T10:53:43.069501: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 49
[2025-01-12T10:53:43.070308: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 50
[2025-01-12T10:53:43.070789: : <36> addPluginMetricCounter@nginx_intaker_metric.cc:267    | ###] Unsupported metric type. Type: 51
[2025-01-12T10:53:43.072031: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:53:44.049932: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:53:44.089786: : <36> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-12T10:57:13.977903: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:57:14.816794: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:57:14.844463: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:57:14.863731: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:57:15.135970: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.19.0.1, Error: Maxmind db is uninitialized
[2025-01-12T10:57:23.026243: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T10:57:23.231631: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T10:58:09.420798: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:58:09.422253: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T10:58:09.423003: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T10:58:09.423758: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:58:09.456179: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T10:58:09.457219: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T10:58:09.458071: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T10:58:09.458873: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T10:58:09.459644: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:58:09.460486: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:58:09.462066: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:58:09.462742: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:58:09.463440: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T10:59:57.386118: : <32> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='TypeIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T11:00:00.006496: : <32> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='', owner='TypeIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T11:00:00.144403: : <32> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Sync worker learning on load
[2025-01-12T11:03:20.173601: : <1> run@mainloop.cc:352                                    | ---] Mainloop ended - stopping all routines
[Sun Jan 12 11:12:49 UTC 2025][addStream@logging.cc:107                                     | ---] Successfully added log stream. Stream type: JSON FOG stream
[init@components_list_impl.h:126                              | ---] Initializing component: Environment
[init@components_list_impl.h:126                              | ---] Initializing component: DebugIS
[2025-01-12T11:12:50.157738: : init@components_list_impl.h:126                            | ---] Initializing component: Version
[2025-01-12T11:12:50.158840: : init@components_list_impl.h:126                            | ---] Initializing component: Buffer
[2025-01-12T11:12:50.162113: : init@components_list_impl.h:126                            | ---] Initializing component: ShellCmd
[2025-01-12T11:12:50.163846: : init@components_list_impl.h:126                            | ---] Initializing component: GenericMetric
[2025-01-12T11:12:50.165711: : init@components_list_impl.h:126                            | ---] Initializing component: Messaging
[2025-01-12T11:12:50.186834: : init@components_list_impl.h:126                            | ---] Initializing component: MainloopComponent
[2025-01-12T11:12:50.188491: : init@components_list_impl.h:126                            | ---] Initializing component: ConfigComponent
[2025-01-12T11:12:50.191884: : init@components_list_impl.h:126                            | ---] Initializing component: InstanceAwareness
[2025-01-12T11:12:50.193069: : init@components_list_impl.h:126                            | ---] Initializing component: IntelligenceComponentV2
[2025-01-12T11:12:50.194024: : init@components_list_impl.h:126                            | ---] Initializing component: AgentDetails
[2025-01-12T11:12:50.217021: : getMachineTypeFromDmiTable@agent_details.cc:354            | ###] Error. Could not decode the DMI table. Table value is empty
[2025-01-12T11:12:50.220174: : registerMachineType@agent_details.cc:376                   | ###] Error. Machine type is unrecognized
[2025-01-12T11:12:50.221481: : registerMachineType@agent_details.cc:381                   | ---] Setting machine type 3
[2025-01-12T11:12:50.229897: : init@components_list_impl.h:126                            | ---] Initializing component: LoggingComp
[2025-01-12T11:12:50.231291: : init@components_list_impl.h:126                            | ---] Initializing component: TimeProxyComponent
[2025-01-12T11:12:50.232190: : init@components_list_impl.h:126                            | ---] Initializing component: SignalHandler
[2025-01-12T11:12:50.234170: : init@components_list_impl.h:126                            | ---] Initializing component: RestServer
[2025-01-12T11:12:50.235096: : init@components_list_impl.h:126                            | ---] Initializing component: Encryptor
[2025-01-12T11:12:50.235985: : init@components_list_impl.h:126                            | ---] Initializing component: SocketIS
[2025-01-12T11:12:50.239410: : init@components_list_impl.h:126                            | ---] Initializing component: CPUCalculator
[2025-01-12T11:12:50.242263: : init@components_list_impl.h:126                            | ---] Initializing component: CPUManager
[2025-01-12T11:12:50.247881: : init@components_list_impl.h:126                            | ---] Initializing component: MemoryCalculator
[2025-01-12T11:12:50.249129: : init@components_list_impl.h:126                            | ---] Initializing component: TenantManager
[2025-01-12T11:12:50.249859: : init@components_list_impl.h:126                            | ---] Initializing component: GenericRulebase
[2025-01-12T11:12:50.251326: : init@components_list_impl.h:126                            | ---] Initializing component: Table
[2025-01-12T11:12:50.252457: : init@components_list_impl.h:126                            | ---] Initializing component: NginxAttachment
[2025-01-12T11:12:50.254011: : init@nginx_attachment.cc:263                               | ---] Successfully initialized NGINX Attachment
[2025-01-12T11:12:50.254781: : init@components_list_impl.h:126                            | ---] Initializing component: GradualDeployment
[2025-01-12T11:12:50.255328: : init@components_list_impl.h:126                            | ---] Initializing component: HttpManager
[2025-01-12T11:12:50.256190: : init@components_list_impl.h:126                            | ---] Initializing component: Layer-7 Access Control
[2025-01-12T11:12:50.258575: : init@components_list_impl.h:126                            | ---] Initializing component: RateLimit
[2025-01-12T11:12:50.260211: : init@components_list_impl.h:126                            | ---] Initializing component: WaapComponent
[2025-01-12T11:12:54.600587: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ScannerDetector'
[2025-01-12T11:12:54.612862: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-12T11:12:54.667326: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T11:12:54.683299: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='ConfidenceCalculator'
[2025-01-12T11:12:54.709595: : SerializeToLocalAndRemoteSyncBase@Serializator.cc:418      | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T11:12:54.864769: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-ab.js, static resource's path: /etc/cp/conf/waap/cp-ab.js
[2025-01-12T11:12:54.866949: : registerStaticResource@nginx_attachment.cc:364             | ###] Failed to write the static resource to the shared memory. Resource name: cp-csrf.js, static resource's path: /etc/cp/conf/waap/cp-csrf.js
[2025-01-12T11:12:54.867481: : init@components_list_impl.h:126                            | ---] Initializing component: IPSComp
[2025-01-12T11:12:54.868563: : init@components_list_impl.h:126                            | ---] Initializing component: KeywordComp
[2025-01-12T11:12:54.870498: : init@components_list_impl.h:126                            | ---] Initializing component: GeoLocation
[2025-01-12T11:12:54.872654: : initGeoDbObj@geo_location.cc:109                           | ###] maxMindDB error: Error opening the specified MaxMind DB file
[2025-01-12T11:12:54.875236: : init@components_list_impl.h:126                            | ---] Initializing component: HttpGeoFilter
[2025-01-12T11:12:54.879983: : run@mainloop.cc:236                                        | ---] Starting the Mainloop
[2025-01-12T11:12:54.881500: : <14> operator()@rest_server.cc:210                         | ---] REST server started: 12004. Accepting: loopback connections
[2025-01-12T11:12:54.930662: : <5> reportStartupEvent@mainloop.cc:224                     | ---] Startup report was successfully sent to fog
[2025-01-12T11:12:54.931927: : <10> closeLogFile@file_stream.cc:119                       | ###] Failed in closing log file. File path: 5
[2025-01-12T11:12:57.179757: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:12:58.596142: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:15:49.929177: : <25> registerAttachmentProcess@nginx_attachment.cc:540     | ---] Successfully registered attachment
[2025-01-12T11:15:49.933891: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:15:49.944345: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ScannerDetector'
[2025-01-12T11:15:49.949662: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-12T11:15:49.965517: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T11:15:49.979492: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='ConfidenceCalculator'
[2025-01-12T11:15:49.989266: : <36> SerializeToLocalAndRemoteSyncBase@Serializator.cc:418 | ---] Create SerializeToLocalAndRemoteSyncBase. assetId='Any', owner='TrustedSourcesConfidenceCalculator'
[2025-01-12T11:15:50.188877: : <36> run@mainloop.cc:317                                   | ###] Routine execution exceeded run time. Routine name: Nginx Attachment inspection handler
[2025-01-12T11:15:50.192219: : <38> getSharedStorageHost@TuningDecision.cc:184            | ---] shared storage host is set to appsec-shared-storage
[2025-01-12T11:15:50.201245: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T11:15:50.215819: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T11:15:50.216762: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:15:50.217504: : <38> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:15:50.218195: : <38> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:15:50.218885: : <38> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:15:50.225489: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T11:15:50.226533: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T11:15:50.228565: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:15:50.229331: : <47> establishNewConnection@connection_comp.cc:104         | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:15:50.229925: : <47> getConnection@messaging_comp.cc:107                   | ###] Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:15:50.232156: : <47> sendMessage@messaging_comp.cc:123                     | ###] Failed to get connection. Error: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:15:50.233723: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -1, error response: Failed to establish connection. Error: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:15:50.234873: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.236305: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.238585: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.241349: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.257317: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.258174: : <40> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T11:15:50.258877: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.259679: : <40> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.260443: : <40> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T11:15:50.261711: : <40> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-12T11:15:50.262461: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.263095: : <42> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T11:15:50.263618: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.264845: : <42> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Indicators/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.265649: : <42> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T11:15:50.266373: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.267203: : <44> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T11:15:50.268031: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.268781: : <44> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Confidence/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.269382: : <44> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T11:15:50.270106: : <44> pullProcessedData@ConfidenceCalculator.cc:230         | !!!] Failed to get the remote state
[2025-01-12T11:15:50.270845: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/remote, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.271838: : <46> getRemoteProcessedFilesList@Serializator.cc:774       | ###] Failed to get the list of files
[2025-01-12T11:15:50.272441: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.273127: : <46> sendObject@i_serialize.h:165                          | ###] failed to send request to uri: /api/?list-type=2&prefix=d62213d2-7dda-4b14-9003-b0a407809f91/Any/5/Type/Trust/processed, error: [Status-code]: -2 - HTTP_SUSPEND, [Body]: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:15:50.278549: : <46> getProcessedFilesList@Serializator.cc:844             | ###] Failed to get the list of files
[2025-01-12T11:15:51.050163: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T11:15:51.219189: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:15:52.049889: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:15:56.377602: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:15:59.509442: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:15:59.832867: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:16:00.062037: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:16:00.125861: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:16:16.657280: : <36> getExceptionVerdict@http_geo_filter.cc:346            | ###] Lookup location failed for source: 172.18.0.1, Error: Maxmind db is uninitialized
[2025-01-12T11:19:59.830311: : <37> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='ScannerDetector' last modified state: 
[2025-01-12T11:19:59.855165: : <37> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T11:19:59.856156: : <37> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T11:19:59.856850: : <37> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:19:59.857598: : <37> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T11:19:59.858695: : <37> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:19:59.859718: : <37> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T11:19:59.860420: : <39> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='KeywordIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T11:19:59.863474: : <39> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:19:59.864311: : <39> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T11:19:59.864927: : <41> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='TrustedSourcesConfidenceCalculator' last modified state: 
[2025-01-12T11:19:59.865674: : <41> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:19:59.866408: : <41> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T11:19:59.867259: : <43> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='TypeIndicatorFilter/ConfidenceCalculator' last modified state: 
[2025-01-12T11:19:59.869445: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:19:59.870471: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T11:19:59.871216: : <45> syncWorker@Serializator.cc:656                        | ---] Running the sync worker for assetId='Any', owner='TrustedSourcesConfidenceCalculator' last modified state: 
[2025-01-12T11:19:59.871917: : <45> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:19:59.873097: : <45> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 0
[2025-01-12T11:20:04.873585: : <37> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:04.874053: : <37> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T11:20:04.874896: : <39> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:04.875241: : <39> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T11:20:04.875796: : <41> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:04.876162: : <41> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T11:20:04.878854: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:04.879342: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T11:20:04.879860: : <45> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:04.880281: : <45> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 1
[2025-01-12T11:20:09.877042: : <37> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:09.878004: : <37> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T11:20:09.879072: : <39> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:09.879472: : <39> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T11:20:09.880119: : <41> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:09.880647: : <41> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T11:20:09.881409: : <43> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:09.881863: : <43> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T11:20:09.882500: : <45> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:20:09.882950: : <45> sendNoReplyObjectWithRetry@i_serialize.h:244          | ---] Failed to send object. Attempt: 2
[2025-01-12T11:20:14.889907: : <37> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/ScannersDetector/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T11:20:14.892072: : <37> postData@ScannerDetector.cc:106                       | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/ScannersDetector/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T11:20:14.898417: : <39> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T11:20:14.899340: : <39> postData@ConfidenceCalculator.cc:143                  | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Confidence/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T11:20:14.904061: : <41> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T11:20:14.905474: : <41> postData@TrustedSourcesConfidence.cc:102              | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Indicators/Trust/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T11:20:14.910660: : <43> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/window_20100_10/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T11:20:14.911860: : <43> postData@ConfidenceCalculator.cc:143                  | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Confidence/window_20100_10/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T11:20:14.915466: : <45> sendNoReplyObjectWithRetry@i_serialize.h:247          | ###] Failed to send object to /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data, reached maximum attempts: 3
[2025-01-12T11:20:14.916371: : <45> postData@TrustedSourcesConfidence.cc:102              | !!!] Failed to post collected data to: /api/d62213d2-7dda-4b14-9003-b0a407809f91/Any/Type/Trust/window_20100_4/e2026b08-ddcb-4512-bbd7-4f0d168a55ed/5/data.data
[2025-01-12T11:20:26.947300: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T11:20:27.163002: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:22:55.699245: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T11:22:55.868748: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
[2025-01-12T11:25:49.822499: : <38> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-shared-storage:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T11:25:49.824105: : <38> connectToHost@connection.cc:521                       | ###] Connection to: appsec-shared-storage:80 failed and will be suspended for 60 seconds
[2025-01-12T11:25:49.824789: : <38> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-shared-storage:80. There won't be a retry attempt.
[2025-01-12T11:25:49.825785: : <38> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T11:25:49.832539: : <47> tryToBioConnect@connection.cc:479                     | ###] Connection to: appsec-tuning-svc:80 failed and won't retry. Error: error:2008F002:BIO routines:BIO_lookup_ex:system lib
[2025-01-12T11:25:49.833563: : <47> connectToHost@connection.cc:521                       | ###] Connection to: appsec-tuning-svc:80 failed and will be suspended for 60 seconds
[2025-01-12T11:25:49.834481: : <47> establishConnection@connection.cc:193                 | ###] Failed to connect to host: appsec-tuning-svc:80. There won't be a retry attempt.
[2025-01-12T11:25:49.835022: : <47> isSuspended@connection.cc:176                         | ###] Reestablish connection failed
[2025-01-12T11:25:49.835553: : <47> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:25:49.836389: : <48> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:25:49.837113: : <49> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:25:49.837732: : <50> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:25:49.838369: : <51> sendSyncMessageWithoutResponse@interface_impl.h:114   | ###] Received error from server. Status code: -2, error response: The connection is suspended due to consecutive message sending errors.
[2025-01-12T11:25:51.553193: : <4> receiveData@connection.cc:582                          | ###] Connection closed by peer
[2025-01-12T11:25:51.760386: : <4> run@mainloop.cc:317                                    | ###] Routine execution exceeded run time. Routine name: Handling in-memory messages
